pyiron/atomistics/job/interactivewrapper.py

```diff
@@ -162,7 +164,11 @@ def db_entry(self):
         """
         db_dict = super(InteractiveWrapper, self).db_entry()
         if self.structure:
-            parent_structure = self.structure.get_parent_basis()
+            if type(self.structure) == ASEAtoms:
```

Comment:
Can you do the comparison like we did it here? 
https://github.com/pyiron/pyiron/blob/master/pyiron/gpaw/gpaw.py#L28

It is recommended to use `isinstance()` rather than `type()` so we check if it is an instance of our atoms class which is by now derived from the ASE atoms class.